app/models/image.rb

```diff
@@ -6,7 +6,10 @@ class Image < ApplicationRecord
   include ShareSupport
 
   has_one :user, inverse_of: :image
-  belongs_to :imageable, polymorphic: true, touch: true
+  belongs_to :item
```

Comment:
@patrixr is this correct?

Reply:
that looks wrong, that would remove the polymorphic relationship altogether

Reply:
@steveyken @patrixr , I havent removed that line
I have just moved it below

The belongs_to side of the relationship needed to be defined before the polymorphic association was defined. Else Rails didnt recognise the relationship and was throwing error

Reply:
got it, thanks